tests/integration/components/flexberry-field-test.js

```diff
@@ -35,12 +35,33 @@ test('it renders properly', function(assert) {
 
   // Retrieve component.
   let $component = this.$().children();
+  let $fieldInput = $component.children('DIV');
```

Comment:
Select as 'DIV' is not enough, because multiple DIV's can exist inside field & there is no warranty that first selected DIV is related to inner flexberry-textbox. So, select flexberry-textbox like that: $component.children('div .flexberry-textbox').